Jenkinsfile.cd

```diff
@@ -278,5 +301,5 @@ testAndPublish(
         ledger: ledgerTestUbuntu,
         stp: stpTestUbuntu]
     ],
-    true, options, [ubuntu: buildDebUbuntu], 'plenum'
+    true, options, [ubuntu: buildDebUbuntuXenial, ubuntu: buildDebUbuntuBionic], 'plenum'
```

Comment:
Is it right that we have a map with 2 the same keys 'ubuntu' and different values? How the needed function will be choosen?